### ISSUE TITLE
Any way to remove injected sources from .html if removing a bower dependency from project?

### ISSUE THREAD

**Issue by toymachiner62** [taptapship/wiredep#227]
Is there any way to use wiredep to remove a dependency from my .html file that has been injected via wiredep?

I'm thinking that if i end up removing a bower dependency that is no longer needed, it'd be nice to use the bower hook preuninstall to remove the files from my .html file that were previously inject if i decide i no longer need a bower dependency.

**Comment by gustafl** [comment]
I was lacking this too, and had a good with it today. I used `gulp-delete-lines` to delete injected dependencies. I made one task for deleting `<link ref="stylesheet">` tags and one for deleting `<script>` tags. Then I added my wiredep task to both the postinstall and preuninstall hooks in my `.bowerrc` file, and made the delete tasks and the wiredep task depend on each other in a serially:

1. delete-styles
2. delete-scripts
3. wiredep

Unfortunately, it didn't work out. Running the delete tasks manually works great, but when I run all three tasks together, nothing is deleted. According to console output, each task completed before the next started, but things were still not deleted in the HTML.

I'm content with the delete tasks I created, and run them manually. Hopefully, `gulp.series()` in Gulp 4 will solve it.
